packages/addons-v5/lib/destroy_addon.js

```diff
@@ -0,0 +1,46 @@
+'use strict'
+
+const cli = require('heroku-cli-util')
+
+module.exports = async function (heroku, addon, force, wait) {
+  const {waitForAddonDeprovisioning} = require('./addons_wait')
+
+  function destroyAddonRequest(force) {
+    return cli.action(
+      `Destroying ${cli.color.addon(addon.name)} on ${cli.color.app(addon.app.name)}`,
+      heroku.delete(`/apps/${addon.app.id}/addons/${addon.id}`, {
+        headers: {'Accept-Expansion': 'plan'},
+        body: {force},
+      }).then(function (addon) {
+        if (addon.state === 'deprovisioning') {
+          cli.action.done(cli.color.yellow('pending'))
+        }
+
+        return addon
+      }).catch(error => {
+        if (error.body && error.body.message) {
+          throw new Error(`The add-on was unable to be destroyed: ${error.body.message}.`)
+        } else {
+          throw new Error(`The add-on was unable to be destroyed: ${error}.`)
+        }
+      }),
+    )
+  }
+
+  addon = await destroyAddonRequest(force)
```

Comment:
Please create a new variable here instead of overwriting the `addon` arg.

Reply:
Done!